internal/server/cookie.go

```diff
@@ -47,13 +47,19 @@ func setCookie(req *http.Request, resp http.ResponseWriter, config cookieConfig)
 	})
 }
 
-func deleteCookie(resp http.ResponseWriter, name, domain string) {
+func deleteCookie(req *http.Request, resp http.ResponseWriter, name, domain string) {
+	secure := true
+	if req.TLS == nil {
```

Comment:
Is this for testing? Or another reason?

Reply:
Assuming that the cookies the client has are `secure` if the request is over `https`, and not otherwise because we need to set the cookie to clear it, and secure cookies cannot be set over http channels. 

`http` is only used during testing locally. Hosted Infra should always be over HTTPs.

Reply:
Ah I see this is simply making this functionality the same as elsewhere, and that makes sense for consistency. LGTM